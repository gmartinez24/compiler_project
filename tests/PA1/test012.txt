// Haden Johnson
// Testing float errors

float 3.14

1. // error

float .1 //  error
1234.356
12.34.45.56
14.14 14.
34 .56
5.5identify