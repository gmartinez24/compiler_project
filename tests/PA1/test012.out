FLOAT
FLOAT_VAL	3.14
ERROR
FLOAT
PERIOD
INT_VAL	1
FLOAT_VAL	1234.356
FLOAT_VAL	12.34
PERIOD
FLOAT_VAL	45.56
FLOAT_VAL	14.14
ERROR
INT_VAL	34
PERIOD
INT_VAL	56
FLOAT_VAL	5.5
IDENT	identify
EOF